src/components/Leaderboards/TotalContribution.re

```diff
@@ -3,7 +3,7 @@ open ReasonApolloHooks;
 module LoadMostContributed = [%graphql
   {|
     query {
-      patronNews(first: 20, orderBy: totalContributed, orderDirection: desc) {
+      patronNews(first: 20, orderBy: totalContributed, orderDirection: desc, where: {id_not: "0x6d47cf86f6a490c6410fc082fd1ad29cf61492d0"}) {
```

Comment:
A hack to remove the steward contract from this list. There is a notion task to fix this issue at the root (in the graph).